src/common/components/giftViews/giftListItem/giftListItem.component.js

```diff
@@ -0,0 +1,31 @@
+import angular from 'angular';
+import template from './giftListItem.tpl';
+import desigSrc from 'common/directives/desigSrc.directive';
+
+let componentName = 'giftListItem';
+
+class GiftListItemController {
+
+  /* @ngInject */
+  constructor() {
+  }
+}
+
+export default angular
+  .module( componentName, [
+    template.name,
+    desigSrc.name
+  ] )
+  .component( componentName, {
+    controller:  GiftListItemController,
+    templateUrl: template.name,
+    transclude:  {
+      'selectInput': '?label'
+    },
+    bindings:    {
+      gift:        '<',
```

Comment:
If angular was really only doing one-way binding, a gift that had _selected added to it in giftListItem should never make it back out into the array of gifts right? I'm guessing it just isn't watching on the scope of the parent but that variable is updated when you specifically read it in the filter in the file you linked. Just seems a little weird.